bytecomp/simplif.ml

```diff
@@ -686,9 +685,9 @@ let split_default_wrapper ?(create_wrapper_body = fun lam -> lam)
         (wrapper_body, (inner_id, inner_fun))
   in
   try
-    let wrapper_body, inner = aux [] body in
-    [(fun_id, Lfunction{kind; params; body = create_wrapper_body wrapper_body;
-       attr; loc}); inner]
+    let body, inner = aux [] body in
+    let attr = default_stub_attribute in
```

Comment:
Is it possible that in the future there may be attributes that we don't want to discard when marking something as a stub?  I wonder if it would be better if there was a function `set_stub_attribute` or something.